README.md

```diff
@@ -0,0 +1,91 @@
+# Flyspray
```

Comment:
try to add badges here
- codacy
- depndabot
- coverage
- github workflow status 
- ...